cmd/dep/status.go

```diff
@@ -432,7 +444,10 @@ func runStatusAll(ctx *dep.Ctx, out outputter, p *dep.Project, sm gps.SourceMana
 		// complete picture of all deps. That eliminates the need for at least
 		// some checks.
 
-		out.BasicHeader()
+		err := out.BasicHeader()
+		if err != nil {
+			return false, 0, err
+		}
```

Comment:
This can be shortened to:
```go
if err := out.BasicHeader(); err != nil {
    return false, 0, err
}
```
Sorry that I didn't notice and suggest this in the first review.

Reply:
Oooh! That's nice and compact!